src/agent/README.md

```diff
@@ -113,3 +113,43 @@ agent control tool, rather than the Kata runtime.
 
 For further details, see the
 [agent control tool documentation](../tools/agent-ctl/README.md#run-the-tool-and-the-agent-in-the-same-environment).
+
+## Agent options
+
+The kata agent has the ability to configure agent options in guest kernel command line at runtime. Currently, the following agent options can be configured: 
+
+| Option | Name | Description  | Type | Default |
+|-|-|:-:|:-:|:-:|
+|`agent.debug_console` | Debug console flag | Allow to connect guest OS running inside hypervisor Connect using `kata-runtime exec <sandbox-id>` | boolean | `false` |
+| `agent.devmode` | Developer mode | Allow the agent process to coredump | boolean | `false` |
+| `agent.trace` | Trace mode | Allow to static tracing | boolean | `false` |
+| `agent.log` | Log level | Allow the agent log level to be changed (produces more or less output) | string | `"info"` |
+| `agent.server_addr` | Server address | Allow the ttRPC server address to be specified | string | `"vsock://-1:1024"` |
+| `agent.passfd_listener_port` | File descriptor passthrough IO listener port | Allow to set the file descriptor passthrough IO listener port | integer | `0` |
+| `agent.hotplug_timeout` | Hotplug timeout | Allow to configure hotplug timeout of block devices | integer | `3` |
```

Comment:
I know you mention in in the note below, but I think it would be helpful to mention that the timeout is in seconds in the description?

Reply:
I agree.

Reply:
Fixed.